### PR TITLE
Migrate ready off sapphire

### PR DESCRIPTION
**Pull request by KevinBacabac**
## Summary of Changes
Migrates ready logic (after startup) to a separate discord.js based file.

## PRs
Resolves https://github.com/uwcsc/codeybot/issues/314

## Steps to Reproduce
Behaviour is confirmed to be the same with no regressions.

### CODE REVIEW

File: src/listeners/ready.ts [hunk 1]
```diff
@@ -1,16 +1,9 @@
 import type { ListenerOptions, Piece, PieceContext } from '@sapphire/framework';
 import { Listener, Store } from '@sapphire/framework';
```
Comment (mcpenguin) [inline src/listeners/ready.ts:2]: Why are we still using Sapphire's listener?

Comment (KevinBacabac) [reply src/listeners/ready.ts]: For the record we removed this file after deciding we don't use the store feature

**Review by Picowchew — APPROVED** [review]
LGTM!

**Review by botatooo — APPROVED** [review]
LGTM

**Pull request merged**